builder-base/install.sh

```diff
@@ -102,13 +102,35 @@ yum install -y \
     vim \
     which
 
-ROOTLESSKIT_VERSION="${ROOTLESSKIT_VERSION:-v0.13.2}"
+# Install image-builder build dependencies
+# Post upgrade, pip3 got renamed to pip and moved locations. It works completely with python3
+# Symlinking pip3 to pip, to have pip3 commands work successfully
+pip3 install -U pip setuptools
+ln -s /usr/local/bin/pip /usr/bin/pip3
```

Comment:
Looking at these paths and I'm getting nervous, perhaps this is not the best approach. The normal Pythonic way of doing things is with virtual environments. Whatever version of Python or pip you have to start, this should work.

```
pip install virtualenv
virtualenv venv
source venv/bin/activate
pip install pip # etc
```